Library/Formula/bar.rb

```diff
@@ -1,11 +1,15 @@
-require 'formula'
-
 class Bar < Formula
-  homepage 'http://www.theiling.de/projects/bar.html'
-  url 'http://www.theiling.de/downloads/bar-1.4-src.tar.bz2'
-  sha1 '00b55ca1d52ca6ed099937692697c2257833c73c'
+  homepage "http://www.theiling.de/projects/bar.html"
+  url "http://www.theiling.de/downloads/bar-1.4-src.tar.bz2"
+  sha1 "00b55ca1d52ca6ed099937692697c2257833c73c"
 
   def install
-    bin.install 'bar'
+    bin.install "bar"
+  end
+
+  test do
+    (testpath/"test1").write "pumpkin"
+    (testpath/"test2").write "latte"
+    assert shell_output("#{bin}/bar test1 test2").include? "latte"
```

Comment:
Thanks. Pushed! My text editor always throws tantrums about ` /latte/ ` in Ruby code. Not sure why.